src/k8s/deployment.ts

```diff
@@ -16,6 +16,14 @@ export class Command {
     static getDeploymentConfigs() {
         return `oc get deploymentConfig -o json`;
     }
+
+    static getReplicas(deploymentConfig: string) {
+        return `oc get rc -o jsonpath="{range .items[?(.metadata.annotations.openshift\\.io/deployment-config\\.name=='${deploymentConfig}')]}{.metadata.name}{\\"\\n\\"}{end}"`;
+    }
+
+    static delete(replica: String) {
+        return `oc delete rc ${replica}`;
```

Comment:
@dgolovin `oc delete` command first delete the latest rc and redeploy the latest rc again with the same name(latest rc).